src/applications/mhv-secure-messaging/tests/containers/App.unit.spec.jsx

```diff
@@ -364,4 +364,24 @@ describe('App', () => {
     expect(queryByText('Messages', { selector: 'h1', exact: true }));
     expect(window.location.replace.called).to.be.true;
   });
+
+  it('displays Page Not Found component if bad url', () => {
+    const screen = renderWithStoreAndRouter(<App />, {
+      initialState: {
+        featureToggles: {
+          // eslint-disable-next-line camelcase

```

Comment:
if you want to avoid this eslint warning, you can declare `intialState` const with `featureToggles` object. and then set `initialState.featureToggels['mhv_secure_messaging_to_va_gov_release']` to `true`